deploy/qgroundcontrol-start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+export LD_LIBRARY_PATH=$HOME/Qt/5.5/gcc_64/lib
```

Comment:
The user may have some other system libraries in the ld path. The proper way to define this would be:
```export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HOME/Qt/5.5/gcc_64/lib```
That way you are *adding* a path rather than replacing it all.